README.md

```diff
@@ -61,6 +61,24 @@ should look something like this:
 If you run `./scripts/lamp -b make` it should set up everything for you.
 You just need to set the "Generation Path" to your `build` directory.
 
+
+## Lint Workload YAML Files
+```bash
+# (Optional) Install Python virtualenv
+python3 -m pip install virtualenv
+python3 -m virtualenv venv
+source venv/bin/activate
+
+# Install Genny Python packages
+cd src/python
+python setup.py install
+cd -
```

Comment:
the above is basically SOP for all of our python scripts - do we repeat this info elsewhere and/or should we refactor to a README within the python dir for how to set it all up?

Reply:
Oh. I missed the readme in the python directory. :(

I'll move this over and add a line to this readme that refers to the one in src/python if people want to parse metrics or lint yaml.

Also, I didn't end up adding the lint-yaml command to lamp because I think it helps the Genny onboarding experience just a bit more if people didn't have to install any packages for lamp